src/examples/meson.build

```diff
@@ -36,6 +36,7 @@ source_file = [
     'InvMasking.cpp',
     'GradientMasking.cpp',
     'AnimateMasking.cpp',
+    'TvgLoader.cpp',
```

Comment:
The list of files should be in ascending alphabetical order.

(Some spies are hiding in this list.)

Reply:
Updated meson.build for ascending alphabetical order